tenable/io/v3/assets/schema.py

```diff
@@ -20,10 +20,16 @@ class AssetSchema(Schema):
     '''
     Asset API Schema
     '''
+    fqdn = fields.List(fields.Str())
+    ipv4 = fields.List(fields.Str())
+    netbios_name = fields.Str()
+    mac_address = fields.List(fields.Str())
+    id = fields.Str()
+
     class Meta:
         unknown = INCLUDE
 
-    @validates_schema
+    @validates_schema(pass_many=True)
```

Comment:
why **pass_many** is set to true, when you are expecting data to be a single dictionary instead of collection/list ?

Reply:
user can provide collection of assets as well, thats why we set pass_many=True